packages/mint-components/src/components/tax-and-cash/sqm-tax-form/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+
```

Comment:
If you haven't already, make sure to install the VScode exstension "Cucumber (Gherkin) Full Support"